{
  "uid" : "a71ffab6bc3bb321",
  "name" : "实名认证",
  "fullName" : "test_case.User.test_realname#test_realname",
  "historyId" : "d6666fcad673fa0ef5e5af13237f96e1",
  "time" : {
    "start" : 1602685046332,
    "stop" : 1602685046332,
    "duration" : 0
  },
  "status" : "broken",
  "statusDetails" : {
    "message" : "AttributeError: 'tuple' object has no attribute 'url'",
    "trace" : "@pytest.fixture(scope='session')\n    def token():\n        url='http://qa.yansl.com:8084/login'\n        req={\n             \"pwd\": \"123456zjc\",\n             \"userName\": \"123zjc\"\n            }\n>       resp = request_tool.post_json(url, json=req)\n\ntest_case\\conftest.py:10: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nargs = ('http://qa.yansl.com:8084/login',)\nkwargs = {'json': {'pwd': '123456zjc', 'userName': '123zjc'}}, r = ()\n\n    def _func(*args, **kwargs):\n        r= func(*args, **kwargs)\n        request = \"-------------------request-------------\" \\\n>                 \"\\n{0}\\n{1}\\n{2}\".format(r.url, string_tool.dic_to_str(r.request.headers), r.request.body)\nE       AttributeError: 'tuple' object has no attribute 'url'\n\ntools\\report\\decorators_tool.py:11: AttributeError",
    "flaky" : false
  },
  "beforeStages" : [ {
    "name" : "token",
    "time" : {
      "start" : 1602685046333,
      "stop" : 1602685051543,
      "duration" : 5210
    },
    "status" : "broken",
    "statusDetails" : {
      "message" : "AttributeError: 'tuple' object has no attribute 'url'\n",
      "trace" : "  File \"E:\\softweardata\\pycharm_data\\pycharm_project\\pytest\\venv\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"E:\\softweardata\\pycharm_data\\pycharm_project\\pytest\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 923, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"E:\\softweardata\\pycharm_data\\pycharm_project\\pytest\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 783, in call_fixture_func\n    res = fixturefunc(**kwargs)\n  File \"E:\\softweardata\\pycharm_data\\pycharm_project\\pytest\\test_case\\conftest.py\", line 10, in token\n    resp = request_tool.post_json(url, json=req)\n  File \"E:\\softweardata\\pycharm_data\\pycharm_project\\pytest\\tools\\report\\decorators_tool.py\", line 11, in _func\n    \"\\n{0}\\n{1}\\n{2}\".format(r.url, string_tool.dic_to_str(r.request.headers), r.request.body)\n",
      "flaky" : false
    },
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : true,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true
  } ],
  "testStage" : {
    "status" : "broken",
    "statusDetails" : {
      "message" : "AttributeError: 'tuple' object has no attribute 'url'",
      "trace" : "@pytest.fixture(scope='session')\n    def token():\n        url='http://qa.yansl.com:8084/login'\n        req={\n             \"pwd\": \"123456zjc\",\n             \"userName\": \"123zjc\"\n            }\n>       resp = request_tool.post_json(url, json=req)\n\ntest_case\\conftest.py:10: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nargs = ('http://qa.yansl.com:8084/login',)\nkwargs = {'json': {'pwd': '123456zjc', 'userName': '123zjc'}}, r = ()\n\n    def _func(*args, **kwargs):\n        r= func(*args, **kwargs)\n        request = \"-------------------request-------------\" \\\n>                 \"\\n{0}\\n{1}\\n{2}\".format(r.url, string_tool.dic_to_str(r.request.headers), r.request.body)\nE       AttributeError: 'tuple' object has no attribute 'url'\n\ntools\\report\\decorators_tool.py:11: AttributeError",
      "flaky" : false
    },
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "7fc4f2b048bd26d8",
      "name" : "log",
      "source" : "7fc4f2b048bd26d8.txt",
      "type" : "text/plain",
      "size" : 402
    } ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : true,
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "parentSuite",
    "value" : "test_case.User"
  }, {
    "name" : "suite",
    "value" : "test_realname"
  }, {
    "name" : "subSuite"
  }, {
    "name" : "host",
    "value" : "DESKTOP-679Q8H2"
  }, {
    "name" : "thread",
    "value" : "4200-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test_case.User.test_realname"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : true,
  "extra" : {
    "tags" : [ ]
  },
  "source" : "a71ffab6bc3bb321.json",
  "statusDetailsSafe" : {
    "message" : "AttributeError: 'tuple' object has no attribute 'url'",
    "trace" : "@pytest.fixture(scope='session')\n    def token():\n        url='http://qa.yansl.com:8084/login'\n        req={\n             \"pwd\": \"123456zjc\",\n             \"userName\": \"123zjc\"\n            }\n>       resp = request_tool.post_json(url, json=req)\n\ntest_case\\conftest.py:10: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nargs = ('http://qa.yansl.com:8084/login',)\nkwargs = {'json': {'pwd': '123456zjc', 'userName': '123zjc'}}, r = ()\n\n    def _func(*args, **kwargs):\n        r= func(*args, **kwargs)\n        request = \"-------------------request-------------\" \\\n>                 \"\\n{0}\\n{1}\\n{2}\".format(r.url, string_tool.dic_to_str(r.request.headers), r.request.body)\nE       AttributeError: 'tuple' object has no attribute 'url'\n\ntools\\report\\decorators_tool.py:11: AttributeError",
    "flaky" : false
  },
  "statusMessage" : {
    "present" : true
  },
  "parameterValues" : [ ]
}